include/api.php

```diff
@@ -3805,14 +3806,14 @@ function api_friendica_notification_seen($type){
 
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
-		if ($a->argc!==4) throw new BadRequestException("Invalid argument count");
+		if (api_user()===false) throw new HTTP\ForbiddenException();
+		if ($a->argc!==4) throw new HTTP\BadRequestException("Invalid argument count");
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `!==` operator.